pom.xml

```diff
@@ -44,7 +44,7 @@
 
 		<!-- Depedency version properties -->
 		<version.httpclient>4.5.5</version.httpclient>
-		<version.openshift-client>4.1.1</version.openshift-client>
+		<version.openshift-client>4.2.0</version.openshift-client>
```

Comment:
4.2.2 is already out, would be nice to upgrade directly to that version

Reply:
Sure, that shouldn't be a problem.